src/transformers/models/t5/modeling_t5.py

```diff
@@ -772,6 +773,30 @@ def forward(
         return outputs  # hidden-states, present_key_value_states, (self-attention position bias), (self-attention weights), (cross-attention position bias), (cross-attention weights)
 
 
+class T5ClassificationHead(nn.Module):
+    """Head for sentence-level classification tasks."""
+
+    def __init__(
+        self,
+        input_dim: int,
+        inner_dim: int,
+        num_classes: int,
+        pooler_dropout: float,
+    ):
```

Comment:
All of the custom model layers and submodules should take a `config` and use that to instantiate itself. I realise this is copied from BART, but all new models should follow this pattern

```suggestion
    def __init__(self, config):
```

Reply:
Sounds good! I'll go ahead and make this change.